<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMedicalRecord</class>
 <widget class="QWidget" name="PageMedicalRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- STYLE LOCAL --- */
QWidget { 
    font-family: &quot;Segoe UI&quot;, sans-serif; 
    background-color: #f0f2f5; 
}

/* Cadres */
QFrame.Card { 
    background-color: white; 
    border-radius: 8px; 
    border: 1px solid #e0e0e0; 
}

/* Onglets */
QTabWidget::pane { border: 0; }
QTabBar::tab { 
    background: #e0e0e0; 
    color: #555; 
    padding: 10px 20px; 
    margin-right: 2px; 
    border-top-left-radius: 5px; 
    border-top-right-radius: 5px; 
    font-weight: bold; 
}
QTabBar::tab:selected { 
    background: white; 
    color: #2d7aed; 
    border-top: 3px solid #2d7aed; 
}

/* --- TEXTES ET INPUTS (NOIR FORC√â) --- */
QLabel, QCheckBox { 
    color: #333333; 
}

QLineEdit, QTextEdit { 
    border: 1px solid #ccc; 
    border-radius: 5px; 
    padding: 8px; 
    background-color: #fafafa; 
    color: #333333; /* Texte Noir */
}
QLineEdit:focus, QTextEdit:focus { 
    border: 1px solid #2d7aed; 
    background-color: white; 
}

/* --- TABLEAUX --- */
QTableWidget {
    background-color: white;
    color: #333333; /* Texte Noir */
    gridline-color: #f0f0f0;
    border: 1px solid #ccc;
    selection-background-color: #e6f0ff;
    selection-color: #2d7aed;
}
QHeaderView::section {
    background-color: #f8f9fa;
    color: #333333; /* Titres Noir */
    padding: 5px;
    border: none;
    border-bottom: 2px solid #2d7aed;
    font-weight: bold;
}

/* --- BOUTONS --- */
QPushButton.ActionBtn { 
    background-color: #2d7aed; 
    color: white; 
    border-radius: 5px; 
    padding: 8px 15px; 
    font-weight: bold; 
    border: none;
}
QPushButton.ActionBtn:hover { background-color: #1a5fc7; }
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_medical">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   
   <item>
    <widget class="QFrame" name="frame_patient_summary">
     <property name="class" stdset="0">
      <string>Card</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #e3f2fd; border: 1px solid #bbdefb;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_patient_info">
      <item>
       <widget class="QLabel" name="label_patient_name">
        <property name="styleSheet">
         <string notr="true">font-size: 18px; font-weight: bold; color: #0d47a1;</string>
        </property>
        <property name="text">
         <string>Patient : (S√©lectionnez un patient)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_patient_allergies">
        <property name="styleSheet">
         <string notr="true">color: #d32f2f; font-weight: bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item>
    <widget class="QTabWidget" name="tabWidget_actions">
     <property name="currentIndex">
      <number>0</number>
     </property>
     
     <widget class="QWidget" name="tab_history">
      <attribute name="title">
       <string>üìú Historique M√©dical</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_history">
       <item>
        <widget class="QTableWidget" name="table_history">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="editTriggers">
          <enum>QAbstractItemView::NoEditTriggers</enum>
         </property>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>M√©decin</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Diagnostic</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_history_details">
         <property name="text">
          <string>D√©tail de la consultation s√©lectionn√©e :</string>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight: bold; margin-top: 10px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_history_view">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Cliquez sur une ligne pour voir le compte-rendu complet...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="subtab_consultation">
      <attribute name="title">
       <string>ü©∫ Nouvelle Consultation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_consult">
       <item>
        <widget class="QLabel" name="lbl_diagnostic">
         <property name="text">
          <string>Diagnostic &amp; Notes :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_diagnostic">
         <property name="placeholderText">
          <string>Saisir les sympt√¥mes, l'observation clinique et le diagnostic...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="subtab_prescription">
      <attribute name="title">
       <string>üíä Ordonnance</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_presc">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_add_med">
         <item>
          <widget class="QLineEdit" name="lineEdit_med_name">
           <property name="placeholderText">
            <string>Nom du m√©dicament</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_med_dose">
           <property name="placeholderText">
            <string>Dosage (ex: 500mg)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_add_med">
           <property name="class" stdset="0">
            <string>ActionBtn</string>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="table_prescription">
         <column>
          <property name="text">
           <string>M√©dicament</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Dosage</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Instructions</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="subtab_report">
      <attribute name="title">
       <string>üìù R√©diger Rapport</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_report">
       <item>
        <widget class="QTextEdit" name="textEdit_report">
         <property name="placeholderText">
          <string>R√©digez ici le compte-rendu officiel de la s√©ance...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="subtab_exams">
      <attribute name="title">
       <string>üî¨ Examens</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_exams">
       <item>
        <widget class="QCheckBox" name="check_blood">
         <property name="text">
          <string>Analyse de sang compl√®te (NFS)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_radio">
         <property name="text">
          <string>Radiographie Thorax</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_mri">
         <property name="text">
          <string>IRM</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_exams">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_validate">
     <property name="topMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_val">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_save_consultation">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #28a745; color: white; border-radius: 5px; font-weight: bold; font-size: 14px; border: none;</string>
       </property>
       <property name="text">
        <string>Terminer la Consultation</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>